meinberlin/assets/scss/components/_formset.scss

```diff
@@ -0,0 +1,54 @@
+.formset--inline .formset__form {
+    display: inline-block;
+    width: 9em;
+    margin-right: 0.5em;
+    margin-bottom: 0.7em;
+    border-bottom: 1px solid $border-color;
+    padding-right: 1.5em;
+    position: relative;
+
+    input {
+        border: 0;
+        margin-bottom: 0;
+        background-color: $body-bg;
+    }
+}
+
+.formset--inline .formset__form--has-error {
+    border-bottom-color: $brand-danger;
+    border-bottom-width: 5px;
+}
+
+.formset__empty {
+    display: none;
+}
+
+.formset__delete {
+    @include sr-only();
+
+    &:focus ~ .formset__delete-label {
+        outline-width: 2px;
+        outline-style: solid;
+        outline-color: Highlight;
+    }
+
+    & ~ .formset__delete-label {
+        position: absolute;
+        right: 0.3em;
+        bottom: 0.3em;
+        color: $gray;
+    }
+
+    &:checked ~ * {
+        opacity: 0.3;
+        pointer-events: none;
+    }
```

Comment:
IMHO this looks very similar to an empty input with a placeholder, which is confusing. I would prefer a better distinction.